lib/index.js

```diff
@@ -25,15 +27,28 @@ function initServer() {
     return server;
 }
 
-function initLogger() {
+function initLogger(abbreviation) {
     const bunyan = require('bunyan');
+    // initialize maximum body logging length
+    bodyAbbreviationLength = abbreviation;
     return bunyan.createLogger({
         name: packageJson.name
     });
 }
 
 function logJson(logger, body) {
-    logger.info(JSON.stringify(body, null, 4));
+    // Make a copy so the original body doesn't get mutated
+    const content = Object.assign({}, body);
```

Comment:
It would be nice to only make a copy when it's actually going to be used (i.e. not for responses with short bodies and not for the call from `makeHandleRequest` where we wouldn't expect it to have a `body` property at all).